lib/notification-issue.coffee

```diff
@@ -183,6 +183,12 @@ class NotificationIssue
         packagePaths[packageName] = fs.realpathSync(packagePath)
 
     getPackageName = (filePath) =>
+      filePath = /(\((.+?):\d+|\((.+)\)|(.+))/.exec(filePath)[1]
```

Comment:
I'm a little confused by the nesting of capture groups here. Does entry `1` in the match array represent the entire match, because the entire thing is wrapped in parens? If so, maybe you could just remove the outer parens and use entry `0`?